utopia-remix/app/routes-test/internal.projects.$id.access.spec.ts

```diff
@@ -0,0 +1,87 @@
+jest.mock('@openfga/sdk')
+import { prisma } from '../db.server'
+import { handleChangeProjectAccess } from '../routes/internal.projects.$id.access'
+import {
+  clearDb,
+  createTestProject,
+  createTestSession,
+  createTestUser,
+  newTestRequest,
+} from '../test-util'
+import { ApiError } from '../util/errors'
+import * as permissionsService from '../services/permissionsService.server'
+import { AccessLevel } from '../types'
+
+describe('handleChangeAccess', () => {
+  afterAll(async () => {
+    jest.restoreAllMocks()
+  })
+  afterEach(async () => {
+    await clearDb(prisma)
+    jest.spyOn(permissionsService, 'setProjectAccess').mockRestore()
+  })
+
+  beforeEach(async () => {
+    await createTestUser(prisma, { id: 'foo' })
+    await createTestUser(prisma, { id: 'bar' })
+    await createTestSession(prisma, { key: 'the-key', userId: 'foo' })
+    await createTestProject(prisma, { id: 'one', ownerId: 'foo', title: 'project-one' })
+    await createTestProject(prisma, { id: 'two', ownerId: 'bar', title: 'project-two' })
+    jest.spyOn(permissionsService, 'setProjectAccess').mockResolvedValue()
+  })
+
+  it('changes access level to private', async () => {
+    const formData = new FormData()
+    formData.append('accessLevel', '0')
```

Comment:
it would be better to use the enum value here rather than hardcoding `0` (same for the other tests below)

Reply:
fixed